README.md

```diff
@@ -33,10 +33,39 @@ in [documementation](https://www.tarantool.io/en/doc/latest/book/box/data_model/
 
     Return values: `true` if no error, otherwise return `nil, err`
 
-Additionally when a section `sharding_key` is not empty in YAML schema call of
-function `ddl.set_schema(schema)` creates a space `_ddl_sharding_key` with two
+Call of function `ddl.set_schema(schema)` creates a space `_ddl_sharding_key` with two
 fields: `space_name` with type `string` and `sharding_key` with type `array`.
 
+Similarly for `sharding_func`: call of function `ddl.set_schema(schema)` creates
+a space `_ddl_sharding_func` with three fields: `space_name` with type `string`,
+`sharding_func_name` with type `string` and `sharding_func_body` with type `string`.
```

Comment:
Are sharding_func_name and sharding_func_body are mutually exclusive?

Reply:
Yes, if user specified sharding function name then information about sharding func is stored in `sharding_func_name` field and `sharding_func_body` is null. If user specified sharding func as table like `{ body = 'function(key) return <...> end' }` then information is stored in `sharding_func_body` field and `sharding_func_name` field is null.